Tasks/Common/packaging-common/locationUtilities.ts

```diff
@@ -150,6 +150,27 @@ export function getWebApiWithProxy(serviceUri: string, accessToken?: string): vs
     return new vsts.WebApi(serviceUri, credentialHandler, options);
 }
 
+// This function is to apply retries generically for any unreliable network calls
+async function retryOnExceptionHelper<T>(action: () => Promise<T>, maxRetries: number, retryIntervalInMilliseconds: number): Promise<T> {
```

Comment:
isn't this maxTries not maxRetries?  because you first decrement the variable before checking < 1.

Reply:
good catch! fixed it